zne/noise_amplification/folding_amplifier/glorious_folding_amplifier.py

```diff
@@ -30,9 +31,37 @@
 class GloriousFoldingAmplifier(DAGNoiseAmplifier):
     """Interface for folding amplifier strategies."""
 
+    def __init__(self, barriers: bool = True, tolerance: float = 0.05) -> None:
+        self._set_barriers(barriers)
+        self._set_tolerance(tolerance)
+
+    ################################################################################
+    ## PROPERTIES
+    ################################################################################
+    @property
+    def barriers(self) -> bool:
+        """Barriers setter"""
+        return self._barriers
+
+    def _set_barriers(self, barriers: bool) -> None:
+        """Set barriers property"""
+        self._barriers = bool(barriers)
+
+    @property
+    def tolerance(self) -> float:
+        """Tolerance setter"""
```

Comment:
Again, not a `setter`, but it is better to explain the property in this docstring. It is somewhat eveident tht this is a `getter` so writing it on the docstring is not adding any relevant information 🙃